resources/migrations/001_update_migrations.yaml

```diff
@@ -5747,17 +5766,6 @@ databaseChangeLog:
                   defaultValue: true
                   remarks: Whether or not to apply formatting to the rows of the export
 
-  - changeSet:
```

Comment:
did you mean to delete this?

Reply:
Should have left a comment, because I do want to delete this. I can't see any reason to keep it, given it is ignored. The reason I deleted it is because the `test-migrations` macro misbehaves with it. The execution of this changeset is included in the `databasechangelog` table which means the `restart-id` [here](https://github.com/metabase/metabase/blob/9cfed2266f056a890746075b8a78b164125d2b7c/test/metabase/db/schema_migrations_test/impl.clj#L248) can be the ID of this changeset, even though it was never actually executed. That meant `(migrate!)` after `(migrate! :down 48)` wasn't actually running the currect migrations in my tests.

Reply:
oh ok, but wonder if this breaks downgrade, see: https://github.com/metabase/metabase/issues/44784#issuecomment-2195483705

I tested and it doesn't seem like it's breaking anything but maybe you want to double check.